pkg/kn/commands/service/service_create.go

```diff
@@ -46,6 +46,10 @@ func NewServiceCreateCommand(p *commands.KnParams) *cobra.Command {
   # Create or replace environment variables of service 's1' using --force flag
   kn service create --force s1 --env KEY1=NEW_VALUE1 --env NEW_KEY2=NEW_VALUE2 --image dev.local/ns/image:v1
 
+  # Create a service with given port
+  # If port is not specified it will take 8080 as a default port
```

Comment:
```
# Create service 'mysvc' with port 80
# If no --port is given the default port 8080 is used
```

I would be more specific and more concise in the documentation.

Reply:
Modified with proper comments